tools/fixFormat.js

```diff
@@ -22,6 +22,7 @@ function fixFormat() {
    *
    */
 
+
```

Comment:
May I ask that this change is for any reason?
A little bit weird that I think this change is not really useful.
Thanks.